host/sgx/report.c

```diff
@@ -145,10 +145,12 @@ static oe_result_t _oe_get_report_internal(
     oe_result_t result = OE_FAILURE;
     oe_report_header_t* header = (oe_report_header_t*)report_buffer;
 
-#if defined(OE_USE_LIBSGX)
+#if defined(OE_HAS_QUOTE_PROVIDER)
     // The two host side attestation API's are oe_get_report and
     // oe_verify_report. Initialize the quote provider in both these APIs.
     OE_CHECK(oe_initialize_quote_provider());
+#else
```

Comment:
This breaks local attestation as well, which doesn't require DCAP libraries. Why are we regressing the functionality here?

Reply:
Nice catch. We should keep local attestation intact.